mail-client/mutt-wizard/mutt-wizard-0_pre20200430.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils
+
+DESCRIPTION="A system for automatically configuring mutt and isync"
+HOMEPAGE="https://github.com/LukeSmithxyz/mutt-wizard"
+
+COMMIT=6fe26400957d9163a75f60fa0c18f266f25fbc85
+SRC_URI="https://github.com/LukeSmithxyz/${PN}/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
+
+S="${WORKDIR}/${PN}-${COMMIT}"
+
+LICENSE="GPL-3"
+SLOT="0"
+
+RDEPEND="
+	app-admin/pass
```

Comment:
this must be at least altered to also provide gopass.

Reply:
Do you mean that there should be a choice?

Reply:
yes, as pass is kinda obsolet thingy and gopass has backward compatibility in anyway.

Reply:
Ah, looked it up, seems like gopass does not add a `pass` command so it is unlikely to work as of now.  
Unless upstream adds an option to check for gopass it can't be added.  
Will create an issue with upstream.

Reply:
upstream issue has been created: https://github.com/LukeSmithxyz/mutt-wizard/issues/433